src/main/java/org/mule/extension/http/api/HttpRequestAttributes.java

```diff
@@ -92,12 +93,23 @@
    */
   @Parameter
   @Optional
-  private final Certificate clientCertificate = null;
+  private transient Certificate clientCertificate;
```

Comment:
how is this value ever obtained? the getter now returns the output of the lazyProvider